core/app/services/uffizzi_core/migrations/account_service.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+class UffizziCore::Migrations::AccountService
+  class << self
+    def remove_draft_accounts
+      UffizziCore::Account.where(state: 'draft').find_each do |account|
+        account.owner.destroy
```

Comment:
Are you sure you want to do that? They have a personal account that they are maybe using.